internal/infrastructure/cache/memcached/cache.go

```diff
@@ -0,0 +1,65 @@
+package memcached
```

Comment:
Why there are two caches: memcached and memory?

Reply:
@vtopc, I forgot to delete it. I thought I'll use for mocking, but haven't written any tests yet :(